src/styles/type-scale/index.md

```diff
@@ -0,0 +1,476 @@
+            }
+          ],
+          [
+            {
+              text: "48"
+            },
+            {
+              html: "<code>govuk-heading-xl<code>"
+            },
+            {
+              text: "32px",
+              format: "numeric"
+            },
+            {
+              text: "35px",
+              format: "numeric"
+            }
+          ],
+          [
+            {
+              text: "36"
+            },
+            {
+              html: "<code>govuk-heading-l<code>"
+            },
+            {
+              text: "27px",
+              format: "numeric"
+            },
+            {
+              text: "30px",
+              format: "numeric"
+            }
+          ],
+          [
+            {
+              text: "27"
+            },
+            {
+              text: "Only used in exceptional circumstances"
+            },
+            {
+              text: "21px",
+              format: "numeric"
+            },
+            {
+              text: "25px",
+              format: "numeric"
+            }
+          ],
+          [
+            {
+              text: "24"
+            },
+            {
+              html: "<code>govuk-heading-m</code>, <code>govuk-body-l</code>"
+            },
+            {
+              text: "21px",
+              format: "numeric"
+            },
+            {
+              text: "25px",
+              format: "numeric"
+            }
+          ],
+          [
+            {
+              text: "19"
+            },
+            {
+              html: "<code>govuk-heading-s</code>, <code>govuk-body</code>"
+            },
+            {
+              text: "19px",
+              format: "numeric"
+            },
+            {
+              text: "25px",
+              format: "numeric"
+            }
+          ],
+          [
+            {
+              text: "16"
+            },
+            {
+              html: "<code>govuk-body-s<code>"
+            },
+            {
+              text: "16px",
+              format: "numeric"
+            },
+            {
+              text: "20px",
+              format: "numeric"
+            }
+          ]
+        ]
+      }) }}
+
+      The scale for large screens has not changed.
+  </div>
+</details>
+
+## Using the type scale in your own CSS
+
+Include the `govuk-font` mixin to use GOV.UK typography in your CSS. 
+
+For example, if you want a custom element to use type scale unit 19, use:
+
+```scss
+@include govuk-font($size: 19);
+``````
+
+You can add additional arguments to control font weight, use tabular font spacing, or to override the line height:
+
+```scss
+@include govuk-font($size 19, $weight: bold, $tabular: true);
+```
+
+### If you only want to set the font size
+
+Do not use `govuk-font` if you only want to change the font size and line height – it includes additional typography-related CSS like the New Transport font family. Instead, you should use `govuk-font-size`:
```

Comment:
Same thing for govuk-font-size (we changed the name so the link to the API docs doesn't exist currently but it will after 5.1 goes out).